packages/framework/src/Component/Elasticsearch/AbstractIndex.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Elasticsearch;
+
+abstract class AbstractIndex
+{
+    protected const BATCH_SIZE = 100;
```

Comment:
Constant BATCH_SIZE is here and in DataProviderInterface. It's confusing to me. Can you explain the reason, please?

Reply:
This one stayed forgotten here. It is not used anywhere. The right place I want to keep it is `DataProviderInterface`.

I am removing it from here.